contracts/Comet.sol

```diff
@@ -1181,6 +1181,9 @@ contract Comet is CometMainInterface {
 
         int104 newPrincipal = principalValue(newBalance);
         updateBasePrincipal(account, accountUser, newPrincipal);
+        if (newBalance > 0) {
+            emit Transfer(address(0), account, uint256(newBalance));
```

Comment:
Wondering if the amount should actually be `presentValueSupply(baseSupplyIndex, newPrincipal)` to protect against any rounding/truncation?

Reply:
Yeah, thinking about it more, we'll need to do that instead of directly using `newBalance` as the amount to protect against rounding down.

Reply:
Modified